CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+cmake_minimum_required (VERSION 3.5)
+
+# Common configuration
+
+set(PACKAGE_VERSION 8.5.0)
```

Comment:
Is the intention that this will match vw versions going forward?

Reply:
this should be the only source

Reply:
this should be the only source

---
In reply to: [190916044](https://github.com/JohnLangford/vowpal_wabbit/pull/1492#discussion_r190916044) [](ancestors = 190916044)